src/modules/reviews/InformationSection/InformationCard/InformationCard.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  informationCardTheme,
+  type InformationCardVariants,
+} from "./InformationCard.theme";
+import type { ReactNode } from "react";
+import { Button } from "@/common/components/Button";
+import { ClipboardIcon } from "@/common/components/CustomIcon";
+import { InformationModal } from "./InformationModal";
+import type { Courses } from "@prisma/client";
+import { InformationCardSkeleton } from "./InformationCardSkeleton";
+
+export type InformationCardProps = InformationCardVariants & {
+  course: Courses;
+  isLocked?: boolean;
+  children: ReactNode;
+};
+
+export const InformationCard = ({
+  course,
+  isLocked,
+  children,
+}: InformationCardProps) => {
+  const { wrapper, header, icon, content, description } =
+    informationCardTheme();
+  return (
+    <div className={wrapper()}>
+      <div className={header()}>
+        <ClipboardIcon className={icon()} />
+        <p>Information</p>
+      </div>
+      <div className={content()}>
+        <div className={description()}>{children}</div>
+        {isLocked ? (
+          <Button as="a" variant="link" href="/auth/login">
```

Comment:
issue: incorrect href to login, should be `/account/auth/login` with a `callbackUrl` query param to the current path

see [`LockCtaOverlay.tsx#L37-L40`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/common/components/LockCtaOverlay/LockCtaOverlay.tsx#L37-L40) for a similar reference

Reply:
Hi @davidlhw, apologise for the critical mistake here. I have fixed this at [eee3712](https://github.com/AfterClass-io/afterclass.io-v2/pull/146/commits/eee3712add2ae853732967c804deb14c766ebdda).

To fix this, I have also refactored the overall structure of the info card. More details can be found in the PR under `Implementation Decisions` Section